public/components/feedback_thumbs.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { EuiButtonIcon, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import React, { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+
+import { UsageCollectionStart } from '../../../../src/plugins/usage_collection/public';
+
+interface Props {
+  appName: string;
+  usageCollection: UsageCollectionStart;
+  className?: string;
+}
+
+export const FeedbackThumbs = ({ usageCollection, appName, className }: Props) => {
+  const [feedback, setFeedback] = useState<'thumbs_up' | 'thumbs_down' | undefined>();
```

Comment:
If visualization is regenerated, will feedback be reset?

Reply:
Yes, it will reset